test/e2e/ingress.go

```diff
@@ -226,26 +465,30 @@ func gcloudUnmarshal(resource, regex, project string, out interface{}) {
 	}
 }
 
-func gcloudDelete(resource, name, project string) {
+func gcloudDelete(resource, name, project string, args ...string) error {
 	framework.Logf("Deleting %v: %v", resource, name)
-	output, err := exec.Command("gcloud", "compute", resource, "delete",
-		name, fmt.Sprintf("--project=%v", project), "-q").CombinedOutput()
+	argList := append([]string{"compute", resource, "delete", name, fmt.Sprintf("--project=%v", project), "-q"}, args...)
+	output, err := exec.Command("gcloud", argList...).CombinedOutput()
 	if err != nil {
```

Comment:
yeah we retry the overall cleanup call https://github.com/kubernetes/kubernetes/pull/26821/files#diff-ec4b96325ef379121837e76bb579c833R198